file_utils.py

```
@@ -6,11 +6,30 @@
+        The filepath of the newly created .cbz file.
+
+    Raises:
+        ValueError: If the inputted filepath does not correspond
+    to a .cbr file.
+
+    Dumps the contents of the .cbr file to a temporary directory
+    and then zips that directory into the .cbz with the same
+    filename.
+    """
     if not get_ext(cbr_path) == ".cbr":
         raise ValueError("Not a .cbr file!")
```

Ruff (0.11.9)

Avoid specifying long messages outside the exception class

(TRY003)